Puzzle.js

```diff
@@ -69,10 +69,16 @@ class Puzzle {
     this.guessedLetters = this.currAnswer.filter((currentIndex) => {	
 			return currentIndex.includes(guess);
 		});
+
+		if (this.guessedLetters.length > 0) {
+			game.currPlayer.updatePlayerScore(wheel.currWheelValue, this.guessedLetters);
```

Comment:
Is this the `game` that your test is complaining about being undefined? I took a look at this whole file, and that would make sense -- game would be undefined because you're not passing it in as an argument to this method. If you want access to the instance of your game in another class, you'll have to pass it in to any methods that need it.